e2e/tasks/commitPerpsOrder.js

```diff
@@ -32,7 +32,7 @@ async function commitPerpsOrder({ wallet, accountId, marketId, sizeDelta, settle
     sizeDelta: ethers.utils.parseEther(`${sizeDelta}`),
     settlementStrategyId,
     acceptablePrice: ethers.utils.parseEther(
-      Math.floor(pythPrice * (sizeDelta > 0 ? 2 : 0.5)).toString()
+      Math.floor(pythPrice * (sizeDelta > 0 ? 1.01 : 0.99)).toString()
```

Comment:
If price moves 1% or you get a bad fill due to skew this will be a bit flakey. Why do we want this tight slippage protection in tests?

Reply:
used for CLI too, not just tests

Reply:
Yea ok, with this tight slippage it would be slightly better to fetch the current fill price based on your size and base the acceptable price on that rather than the raw pyth price

Reply:
what is the slippage to use? 5% is reasonable?

Reply:
yea that's probably safer, just want to avoid flakiness